crates/acir/src/circuit/mod.rs

```diff
@@ -69,9 +89,37 @@ mod test {
         };
 
         let json = serde_json::to_string_pretty(&circuit).unwrap();
-        println!("serialized: {json}");
+        println!("serialized: {}", json);
```

Comment:
Why change this line? `"serialized: {json}"` should be a valid format string

Reply:
does it give a compiler error or print `serialized: {json}` literally at runtime? What version of rustc does the mainframe use? I ask because named format arguments have been standard for quite a while at least, and a raw `{` in a format string not recognized as a format option should give an error. Since you need to write `{{` to print a single { in a format string.

Reply:
I did find a note on it at: https://blog.rust-lang.org/2022/01/13/Rust-1.58.0.html

> This feature works in all macros accepting format strings. However, one corner case is the panic! macro in 2015 and 2018 editions, where panic!("{ident}") is still treated as an unformatted string -- the compiler will warn about this not having the intended effect. Due to the 2021 edition's update of panic macros for [improved consistency](https://doc.rust-lang.org/stable/edition-guide/rust-2021/panic-macro-consistency.html), this works as expected in 2021 panic!.

Reply:
Rust is version 1.58.0-nightly (c390d69a6 2021-10-28)
It is an error, here is an error I have:
error: there is no argument named `crate_name`
```
   --> crates/noirc_frontend/src/hir/resolution/import.rs:162:73
    |
162 |         .unwrap_or_else(|| panic!("error reporter: could not find crate {crate_name}"));
```
This happens for a panic!, but I also have the error with println!